### PR TITLE
Fix bad checksum for ppx_rapper 3.1.0

### PR DESCRIPTION
**Pull request by smorimoto**

### CODE REVIEW

File: packages/ppx_rapper/ppx_rapper.3.1.0/opam [hunk 1]
```diff
@@ -33,7 +33,7 @@ dev-repo: "git+https://github.com/roddyyaga/ppx_rapper.git"
 url {
   src: "https://github.com/roddyyaga/ppx_rapper/archive/3.1.0.tar.gz"
   checksum: [
-    "md5=d6ce377b4a0d8c1c8f9fefd643c35282"
```
Comment (mseri) [inline packages/ppx_rapper/ppx_rapper.3.1.0/opam:36]: Thanks. We should check if the old tarball is in the cache and either use that or check if anything has changed before this is merged. I will have a look at it asap

Comment (smorimoto) [reply packages/ppx_rapper/ppx_rapper.3.1.0/opam:36]: Thank you so much 🙏

**Comment by smorimoto** [comment]
I'm afraid the source may have changed. It fails to build.

```
#16 204.2 [ERROR] The compilation of ppx_rapper failed at "/home/opam/.opam/4.14/bin/dune build -p ppx_rapper -j 31 @install".
#16 204.2 
#16 204.2 #=== ERROR while compiling ppx_rapper.3.1.0 ===================================#
#16 204.2 # context              2.0.10 | linux/x86_64 | ocaml-options-only-flambda.1 ocaml-variants.4.14.1+options | git+https://github.com/ocaml/opam-repository.git
#16 204.2 # path                 ~/.opam/4.14/.opam-switch/build/ppx_rapper.3.1.0
#16 204.2 # command              ~/.opam/4.14/bin/dune build -p ppx_rapper -j 31 @install
#16 204.2 # exit-code            1
#16 204.2 # env-file             ~/.opam/log/ppx_rapper-8-e96b2b.env
#16 204.2 # output-file          ~/.opam/log/ppx_rapper-8-e96b2b.out
#16 204.2 ### output ###
#16 204.2 # (cd _build/default && /home/opam/.opam/4.14/bin/ocamlc.opt -w -40 -g -bin-annot -I lib-runtime/.rapper.objs/byte -I /home/opam/.opam/4.14/lib/angstrom -I /home/opam/.opam/4.14/lib/bigstringaf -I /home/opam/.opam/4.14/lib/caqti -I /home/opam/.opam/4.14/lib/logs -I /home/opam/.opam/4.14/lib/ptime -I /home/opam/.opam/4.14/lib/stringext -I /home/opam/.opam/4.14/lib/uri -no-alias-deps -o lib-runtime/.rapper.objs/byte/rapper.cmi -c -intf lib-runtime/rapper.mli)
#16 204.2 # File "lib-runtime/rapper.mli", line 29, characters 18-36:
#16 204.2 # 29 |   module Stream : Caqti_stream_sig.S with type 'a future := 'a t
#16 204.2 #                        ^^^^^^^^^^^^^^^^^^
#16 204.2 # Error: Unbound module Caqti_stream_sig
```

CC: @roddyyaga

**Comment by mseri** [comment]
Unfortunately it is not in the cache and from https://github.com/roddyyaga/ppx_rapper/tags it looks like the tag was written 4 days ago, probably overwriting the old one. @roddyyaga could you make a new release?

**Comment by mseri** [comment]
Looking at the history, the original 3.1.0 release was tagged at https://github.com/roddyyaga/ppx_rapper/commit/66c198a011798923eb937fbb31771185455d4c6c (so the tarball should be https://codeload.github.com/roddyyaga/ppx_rapper/tar.gz/66c198a011798923eb937fbb31771185455d4c6c)

**Pull request closed without merging**